backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestResultUpload.java

```diff
@@ -0,0 +1,67 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.annotations.Type;
+
+@Getter
+@Setter
+@Entity
+@Slf4j
+@Table(name = "upload")
+public class TestResultUpload extends AuditedEntity {
```

Comment:
I did not know `EternalAuditedEntity` was a thing!

Reply:
I dont think we want to ever delete a record in this table, since its there for historical purposes. Orgs use this history to prove that their CLIA is still in use and they get to keep it active as the RS team shared with us.